nms/app/components/Main.js

```diff
@@ -79,20 +71,48 @@ function Main() {
   );
 }
 
-function NoNetworkFallback() {
+function NoNetworkFallback({networkIds}: {networkIds: Array<string>}) {
+  const location = useLocation();
+  if (networkIds.length > 0) {
+    return (
+      <Navigate to={`/nms/${networkIds[0]}${location.pathname}`} replace />
+    );
+  }
+
   return (
     <AppContextProvider>
       <IndexWithoutNetwork />
     </AppContextProvider>
   );
 }
 
-export default () => (
-  <ApplicationMain>
+export default () => {
```

Comment:
I must say having admin and settings under `/nms/{network_Id}` is rather annoying 😕